app/assets/stylesheets/application.scss

```diff
@@ -2,6 +2,8 @@
 
 @import "./components/miller-columns";
 @import "./components/autocomplete";
+@import "./admin_legacy/govspeak-help";
```

Comment:
Hey - if we need anything from the legacy, we should copy it into `app/assets/stylesheets/admin/components/_govspeak-help.scss`

Reply:
No, none of the legacy stuff in this file is still needed for Design System.

Reply:
ah great so we don't need to import it then?